### DISCUSSION TITLE
怎么将当前帧的画布提取出画面啊？

### DISCUSSION THREAD

**Discussion by qianyunrumo** [Tencent/libpag#2578] [Q&A] [answered]
我使用本地canvas的toDataURL和toBlob返回的画面都是空白的，pag有什么方法可以拿到画面吗

**Comment by kevingpqi123** [comment, marked as answer]
可以看下 PAGSurface 的相关接口，当前帧渲染完成后，可以通过相关接口读取到 RGBA 数据或平台层的图像封装数据
